### ISSUE TITLE
Date selection

### ISSUE THREAD

**Issue by yephny** [qw123wh/Aospdeskclock#16]
I sometimes need to set an alarm for a specific date but it isn't supported in this app. I'd like to, for example, have an alarm to wake me up at 6 a.m. on the 21st.

**Comment by qw123wh** [comment]
hello currently it is possible to program the alarm for a week, however pass on this other updated project
https://github.com/qw123wh/new-clock

**Comment by yephny** [comment]
@qw123wh Are you planning to continue working on the app in a different project called New Clock?

**Comment by qw123wh** [comment]
yes the main project will be new-clock, I released the apk on github you can try it already
